terraform/lambda.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_lambda_function" "service_lambda" {
```

Comment:
@jfromme have a look at this for reference: https://github.com/Pennsieve/integration-service/blob/511660d75adff93f29847e372106390ef8bec27d/terraform/lambda.tf#L29

We had to make some updates for the Go service (2.x) to run on AWS.